### PR TITLE
GIX-1760: Fix spacings nns neuron detail

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

The different sections of NnsNeuronDetails page had the vertical spacing set in different places. This made it easy to have a different spacing between them.

In this PR, all the spacing is set in the parent and the children have no vertical spacing.

# Changes

* Add `noMargin` prop to CardInfo.
* Add `noMargin` prop to SkeletonCard.
* All sections and separators in NnsNeuronDetails are siblings in the html. Just like it looks like in the UI.
* Use the `noMargin` prop and remove the separator in NeuronFollowingCard, NeuronProposalsCard, NeuronVotingHistoryCard and NnsNeuronHotkeysCard.
* Moved the testnet only MakeProposalsCard above the NeuronVotingHistoryCard to have the beginning like the mainnet. I didn't move it below NeuronVotingHistoryCard because this one has an infinite scroll.

# Tests

Only CSS changes.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary. No UI nor UX change.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Same UI:

nns-neuron-details

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/components/ui/SkeletonCard.svelte [hunk 1]
```diff
@@ -2,42 +2,38 @@
...
+    <div class="small" slot="end">
       <SkeletonText />
-    {/if}
-
-    {#if size === "large"}
+    </div>
+    <SkeletonCardContent {size} />
+  </CardInfo>
+{:else}
+  <Card testId="skeleton-card">
```
Comment (dskloetd) [inline frontend/src/lib/components/ui/SkeletonCard.svelte:27]: Does `Card` already have no margin or will `noMargin` just silently not work when `cardType !== "info"`?

File: frontend/src/lib/components/ui/SkeletonCard.svelte [hunk 2]
```diff
@@ -2,42 +2,38 @@
...
+  <CardInfo {noMargin} testId="skeleton-card">
+    <div class="small" slot="start">
       <SkeletonText />
+    </div>
+    <div class="small" slot="end">
       <SkeletonText />
-    {/if}
-
-    {#if size === "large"}
+    </div>
```
Comment (dskloetd) [inline frontend/src/lib/components/ui/SkeletonCard.svelte:23]: Why are the other `div`s not also part of the `SkeletonCardContent`?

File: frontend/src/lib/components/ui/SkeletonCard.svelte [hunk 3]
```diff
@@ -2,42 +2,38 @@
...
-  </div>
-  <div class="small" slot="end">
-    <SkeletonText />
-  </div>
-  <div class="content">
-    <SkeletonText />
-    <SkeletonText />
-
-    {#if ["large", "medium"].includes(size)}
+<!-- We need to explictly use the `Card` or `CardInfo` because props are different -->
```
Comment (dskloetd) [inline frontend/src/lib/components/ui/SkeletonCard.svelte:15]: Why are the props different now?

File: frontend/src/lib/components/ui/SkeletonCard.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/components/ui/SkeletonCard.svelte:27]: `Card` has no margin.

File: frontend/src/lib/components/ui/SkeletonCard.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/lib/components/ui/SkeletonCard.svelte:23]: Because they are in other slots of the Card. The end and start slots.

**Comment by lmuntaner** [comment]
@dskloetd ready for review

File: frontend/src/lib/components/ui/SkeletonCardContent.svelte [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  import { SkeletonText } from "@dfinity/gix-components";
+
+  export let size: "small" | "medium" | "large" = "small";
+</script>
+
+<div class="content">
+  <SkeletonText />
+  <SkeletonText />
+
+  {#if ["large", "medium"].includes(size)}
```
Comment (dskloetd) [inline frontend/src/lib/components/ui/SkeletonCardContent.svelte:11]: `size !== "small"` seems simpler?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/components/ui/SkeletonCard.svelte (hunk 3 above)
Comment (lmuntaner) [reply frontend/src/lib/components/ui/SkeletonCard.svelte:15]: I agree. Makes sense to me too.

File: frontend/src/lib/components/ui/SkeletonCardContent.svelte (hunk 4 above)
Comment (lmuntaner) [reply frontend/src/lib/components/ui/SkeletonCardContent.svelte:11]: I don't want to change implementation in this PR.  Maybe there was a reason for doing it this way.

**Pull request merged**